src/content/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent.mdx

```diff
@@ -21,142 +21,153 @@ signupBanner:
 
-When [enabling browser monitoring](#enable), you can manually insert the JavaScript snippet into your app's webpages. The copy/paste option gives you control over the exact placement of our JavaScript snippet, which is required to monitor the webpage's performance. This is useful for:
+When it comes to installing the browser agent, you've got two options:
 
-* Standalone apps, static sites, and cached pages delivered by CDN
-* <InlinePopover type="apm" /> apps that are not as closely coupled to the browser app as with a standard server-side app (for example, when your client-side app talks to a REST API back end)
+* Option 1: Auto-instrument via a New Relic APM agent (recommended for non-SPA apps)
+  * If you're using an <InlinePopover type="apm" /> (application performance monitoring) agent to monitor the backend-side of your browser app, then you should use the APM injection option. We recommend this option for non-SPA apps because the APM agent will automatically deploy the browser agent to every page of your website, you'll automatically get the latest version of the browser agent whenever you restart your APM agent, and you'll see how backend performance influences the frontend.
+* Option 2: Manually copy and paste the Javascript snippet (recommended for SPA apps)
+  * Use this method if you can't use the first option, due to reasons like you have a SPA app or your APM agent isn't supported for auto-injection ( Node.js, Java agents).
 
-To learn how to use the copy/paste method, watch this short YouTube video (approx. 1:50 minutes).
+In your New Relic UI, select an option and move on to the next step.
 
-<Video
-  id="blPC5R5l7C8"
-  type="youtube"
+<img
+  title="choose a deployment method"
+  alt="choose a deployment method"
+  src={browserInstallStep2}
+  style={{ maxWidth: '70%' }}
 />
 
-Some tips for using the JavaScript snippet:
+</Step>
 
-* **Placement in your webpage:** Copy the code snippet, then paste it inline into your pages as close to the top of the `<head>` element as possible, but after any position-sensitive `<meta>` tags (for example, `X-UA-Compatible` or `charset` information). For more information on the inline `head` placement, see [JavaScript placement requirements](/docs/browser/new-relic-browser/page-load-timing-resources/instrumentation-browser-monitoring#javascript-placement).
-* **License key and app ID:** Near the bottom of the generated JavaScript is your browser license key and application ID. This is useful with the [REST API](/docs/apis/rest-api-v2/browser-examples-v2) and [API Explorer](/docs/apis/rest-api-v2/api-explorer-v2/getting-started-new-relics-api-explorer).
+<Step>
+### (Optional) Enable distributed tracing
 
-### Instrument manually on an APM-monitored app [#agent-instrumentation]
+Next, you can choose to enable distributed tracing. Read more about distributed tracing [here](/docs/browser/new-relic-browser/browser-pro-features/browser-data-distributed-tracing/).
```

Comment:
I'd like a little more in-context guidance about what I should choose here, even if it's just to say: We recommend everyone try out this fantastic doodad. If it's fully optional, why are we including it? We must want people to choose it for some reason :)